### PR TITLE
researcher view with csv download functionality

### PR DESCRIPTION
**Pull request by evmiguel**
[DEMO](https://drive.google.com/file/d/164HsCs7TanIAHm4IDuP4ap7nsAIX8t3i/view?usp=sharing)

### CODE REVIEW

File: client/components/Researcher/index.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        )).json();
+
+        // TODO: this can be dynamically made. Need to figure out what happens when the data is empty
+        const fields = [
+            'response_id',
+            'run_id',
+            'response',
+            'created_at',
+            'ended_at',
+            'response_id',
```
Comment (rwaldron) [inline client/components/Researcher/index.jsx]: `response_id` is a duplicate here

File: client/components/Researcher/index.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+        const csvFile = new Blob([csv], { type: 'text/csv' });
+        const csvURL = URL.createObjectURL(csvFile);
+        const tempLink = document.createElement('a');
+        tempLink.href = csvURL;
+        tempLink.setAttribute(
+            'download',
+            `${encodeURI(this.state.scenarioTitle)}.csv`
+        );
+        tempLink.click();
```
Comment (rwaldron) [inline client/components/Researcher/index.jsx]: Very cool.

File: client/components/ScenariosList/index.jsx [hunk 3]
```diff
@@ -157,7 +141,8 @@ ScenariosList.propTypes = {
 
 function mapStateToProps(state) {
     const { isLoggedIn, username } = state.login;
-    return { isLoggedIn, username };
+    const { scenarios } = state.scenario;
+    return { isLoggedIn, username, scenarios };
```
Comment (rwaldron) [inline client/components/ScenariosList/index.jsx]: I don't quite understand this change

File: client/routes/index.jsx [hunk 4]
```diff
@@ -9,12 +12,53 @@ import Session from '@client/util/session';
...
+
+    async componentDidMount() {
+        const scenarios = await this.getScenarios();
+        if (scenarios) {
+            this.props.setScenarios(scenarios);
+        }
+    }
+
+    async getScenarios() {
+        const scenariosResponse = await (await fetch('api/scenarios')).json();
```
Comment (rwaldron) [inline client/routes/index.jsx]: `scenariosResponse` => `{ scenarios, status }`

File: client/routes/index.jsx [hunk 5]
```diff
@@ -9,12 +12,53 @@ import Session from '@client/util/session';
...
+
+        if (scenariosResponse.status === 200) {
+            return { scenarios: scenariosResponse.scenarios };
```
Comment (rwaldron) [inline client/routes/index.jsx]: And then, with the change above, this becomes: 

```js
        if (status === 200) {
            return { scenarios };
```

File: client/components/ScenariosList/index.jsx (hunk 3 above)
Comment (evmiguel) [reply client/components/ScenariosList/index.jsx]: I'm getting another piece of state called `scenarios` from `state.scenarios`, returned by the Redux reducer

**Pull request merged**